testing/extracttest/extracttest.go

```diff
@@ -27,12 +28,28 @@ type ScanInputMockConfig struct {
 	FakeFileInfo *fakefs.FakeFileInfo
 }
 
+// TestTableEntry is a entry to pass to ExtractionTester
 type TestTableEntry struct {
-	Name              string
-	InputConfig       ScanInputMockConfig
-	WantInventory     []*extractor.Inventory
-	WantErrIs         error
-	WantErrContaining string
+	Name          string
+	InputConfig   ScanInputMockConfig
+	WantInventory []*extractor.Inventory
+	WantErr       error
+}
+
+type String = string
```

Comment:
I'm not sure why we need this - can you add a comment that explains it?

Reply:
Removed, accidentally left this in, I was experimenting with different ways to have ContainsErrStr type.